kubecontrolplane/v1/types.go

```diff
@@ -80,7 +80,7 @@ type KubeAPIServerImagePolicyConfig struct {
 	// registry. The external hostname should be set only when the image registry
 	// is exposed externally. The value is used in 'publicDockerImageRepository'
 	// field in ImageStreams. The value must be in "hostname[:port]" format.
-	ExternalRegistryHostname string `json:"externalRegistryHostname"`
+	ExternalRegistryHostnames []string `json:"externalRegistryHostnames"`
```

Comment:
you can make this change at the same time if you like, but it means you need an origin PR green and read to go to handle it.

Reply:
yeah i'm working on the origin PR.